tasks/01-call-external-alert.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<head>
+ <meta charset="utf-8">
```

Comment:
meta tag should be autoclosed -> `<meta cha..... />`

Reply:
Please create `template.html` in the root folder to have a common html file for tasks in future